linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_profile/ansible/shared.yml

```diff
@@ -5,24 +5,37 @@
 # disruption = low
 {{{ ansible_instantiate_variables("var_accounts_user_umask") }}}
 
-- name: Check if umask is already set
+- name: '{{{ rule_title }}} - Locate profile configuration files where umask is defined'
```

Comment:
Per the [style guide](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible) this should be in title case.

Reply:
Sure. Copy and paste issue! : (

Reply:
fixed